### PR TITLE
Add date_cancelled to user

### PR DESCRIPTION
**Pull request by tdelatorre**
How to test:
- [x] check the code
- [x] go to branch and migrate
- [x] run Taiga in your local
- [x] create a test account in Taiga
- [x] check this new account in db (is_active=true and date_cancelled is empty)
- [x] delete this account in Taiga
- [x] check in db again (now, is_active=false and date_cancelled is filled and is correct)

Thanks for your time! :)

### CODE REVIEW

File: tests/integration/test_users.py [hunk 1]
```diff
@@ -274,6 +275,23 @@ def test_delete_self_user(client):
...
+    client.login(user)
+    response = client.delete(url)
+
+    assert response.status_code == 204
+    user = models.User.objects.get(pk=user.id)
+    assert user.full_name == "Deleted user"
+
+    date_cancelled = datetime.datetime(user.date_cancelled.year, user.date_cancelled.month, user.date_cancelled.day)
+    datetime_now = datetime.datetime.now()
+    date_now = datetime.datetime(datetime_now.year, datetime_now.month, datetime_now.day)
```
Comment (yamila-moreno) [inline tests/integration/test_users.py]: these could be `date_now = datetime.date.today()`

File: tests/integration/test_users.py [hunk 2]
```diff
@@ -344,6 +362,24 @@ def test_cancel_self_user_with_invalid_token(client):
...
+    client.login(user)
+    response = client.post(url, json.dumps(data), content_type="application/json")
+
+    assert response.status_code == 204
+    user = models.User.objects.get(pk=user.id)
+    assert user.full_name == "Deleted user"
+
+    date_cancelled = datetime.datetime(user.date_cancelled.year, user.date_cancelled.month, user.date_cancelled.day)
+    datetime_now = datetime.datetime.now()
+    date_now = datetime.datetime(datetime_now.year, datetime_now.month, datetime_now.day)
```
Comment (yamila-moreno) [inline tests/integration/test_users.py]: these could be `date_now = datetime.date.today()`

**Review by yamila-moreno — CHANGES_REQUESTED** [review]

File: tests/integration/test_users.py (hunk 1 above)
Comment (tdelatorre) [reply tests/integration/test_users.py]: Done!

File: tests/integration/test_users.py (hunk 2 above)
Comment (tdelatorre) [reply tests/integration/test_users.py]: Done!

**Review by yamila-moreno — APPROVED** [review]

**Pull request merged**